README.md

```diff
@@ -4,14 +4,33 @@
 
 > In this project, I will create a personal portfolio. A portfolio website is a key job searching tool for a web developer. In this project, I will precisely replicate one of the *portfolio templates* from Microverse Figma. 
 
-![screenshot](./images/app_screenshot.png)
+![screenshot](./images/Banner.png)
+![screenshot](./images/Contact.png)
+
+This project is part of the Microverse curriculum. The main objective is to understand how to parse a Figma design and follow professional guidlines in responsive design.
+
 
-Additional description about the project and its features.
 
 ## Built With
 
 - Html/Css
 
+## Additional Tools
+
+- Figma
+- stylelint
+
+## Getting Started
+
+To get a local copy of this project:
+
+Clone this repository or download the Zip folder:
+```
+git clone git@github.com:KDlamini/portfolio-website.git
+```
+## Live Demo
+
+[Preview](https://kdlamini.github.io/portfolio-website/)
 
 ## Authors
```

Comment:
Your invitation to go to the `issue page` is missing, please don't forget that, to make a more professional README file.
Example: 
```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!
Feel free to check the [issues page](https://github.com/Your_Github_Account/Repository_Name/issues).
```
- [ ] Add the link to go to your `issue page` in your repository.